boards/lora-e5-dev/include/board.h

```diff
@@ -82,6 +82,27 @@ extern void lora_e5_dev_sx126x_set_rf_mode(sx126x_t *dev, sx126x_rf_mode_t rf_mo
 #define FE_CTRL2            GPIO_PIN(PORT_A, 5)
 /** @} */
 
+/**
+ * @brief    Enable 3.3V output
+ */
+#ifndef CONFIG_LORA_E5_DEV_ENABLE_3p3V
```

Comment:
Should the config symbol only use capital letter ?

```suggestion
#ifndef CONFIG_LORA_E5_DEV_ENABLE_3P3V
```